en/content_moderation_practices.md

```diff
@@ -0,0 +1,48 @@
+# Content Moderation @ Mozilla
```

Comment:
Do we want `@` instead of `at` in a legal document?

Also, do we want a date and a version, e.g. 

```
Version 1.0, effective February 22, 2024
{: datetime="2024-02-22" }
```

Reply:
Let's go ahead and update to "at"

Reply:
@aroybalreid 
Can you tell us if we want to have a version and date? If yes, I can add them and we can merge this one out of the way.